img/h/huggingface.co.svg

```diff
@@ -0,0 +1 @@
-2.66-2.84-5.62-6.25-5.62a5.7 5.7 0 0 0-4.6 2.46c-1-1.26-1.98-2.25-2.86-2.82A7.4 7.4 0 0 0 17.52 48Zm0 4c.51 0 1.14.22 1.82.65 2.14 1.36 6.25 8.43 7.76 11.18.5.92 1.37 1.31 2.14 1.31 1.55 0 2.75-1.53.15-3.48-3.92-2.93-2.55-7.72-.68-8.01.08-.02.17-.02.24-.02 1.7 0 2.45 2.93 2.45 2.93s2.2 5.52 5.98 9.3c3.77 3.77 3.97 6.8 1.22 10.83-1.88 2.75-5.47 3.58-9.16 3.58-3.81 0-7.73-.9-9.92-1.46-.11-.03-13.45-3.8-11.76-7 .28-.54.75-.76 1.34-.76 2.38 0 6.7 3.54 8.57 3.54.41 0 .7-.17.83-.6.79-2.85-12.06-4.05-10.98-8.17.2-.73.71-1.02 1.44-1.02 3.14 0 10.2 5.53 11.68 5.53.11 0 .2-.03.24-.1.74-1.2.33-2.04-4.9-5.2-5.21-3.16-8.88-5.06-6.8-7.33.24-.26.58-.38 1-.38 3.17 0 10.66 6.82 10.66 6.82s2.02 2.1 3.25 2.1c.28 0 .52-.1.68-.38.86-1.46-8.06-8.22-8.56-11.01-.34-1.9.24-2.85 1.31-2.85Z"/><path fill="#FFD21E" d="M38.6 76.69c2.75-4.04 2.55-7.07-1.22-10.84-3.78-3.77-5.98-9.3-5.98-9.3s-.82-3.2-2.69-2.9c-1.87.3-3.24 5.08.68 8.01 3.91 2.93-.78 4.92-2.29 2.17-1.5-2.75-5.62-9.82-7.76-11.18-2.13-1.35-3.63-.6-3.13 2.2.5 2.79 9.43 9.55 8.56 11-.87 1.47-3.93-1.71-3.93-1.71s-9.57-8.71-11.66-6.44c-2.08 2.27 1.59 4.17 6.8 7.33 5.23 3.16 5.64 4 4.9 5.2-.75 1.2-12.28-8.53-13.36-4.4-1.08 4.11 11.77 5.3 10.98 8.15-.8 2.85-9.06-5.38-10.74-2.18-1.7 3.21 11.65 6.98 11.76 7.01 4.3 1.12 15.25 3.49 19.08-2.12Z"/><path fill="#FF9D0B" d="M77.4 48c1.62 0 3.07.66 4.07 1.87a5.97 5.97 0 0 1 1.33 3.76 7.1 7.1 0 0 1 1.95-.3c1.55 0 2.95.59 3.94 1.66a5.8 5.8 0 0 1 .8 7 5.3 5.3 0 0 1 1.78 2.82c.24.9.48 2.8-.8 4.74a5.22 5.22 0 0 1 .37 5.02c-1.02 2.32-3.57 4.14-8.51 6.1-3.08 1.22-5.9 2-5.92 2.01a44.33 44.33 0 0 1-10.93 1.6c-5.86 0-10.05-1.8-12.46-5.34-3.88-5.69-3.33-10.9 1.7-15.92 2.78-2.78 4.63-6.87 5.01-7.77.78-2.66 2.83-5.62 6.24-5.62a5.7 5.7 0 0 1 4.6 2.46c1-1.26 1.98-2.25 2.87-2.82A7.4 7.4 0 0 1 77.4 48Zm0 4c-.51 0-1.13.22-1.82.65-2.13 1.36-6.25 8.43-7.76 11.18a2.43 2.43 0 0 1-2.14 1.31c-1.54 0-2.75-1.53-.14-3.48 3.91-2.93 2.54-7.72.67-8.01a1.54 1.54 0 0 0-.24-.02c-1.7 0-2.45 2.93-2.45 2.93s-2.2 5.52-5.97 9.3c-3.78 3.77-3.98 6.8-1.22 10.83 1.87 2.75 5.47 3.58 9.15 3.58 3.82 0 7.73-.9 9.93-1.46.1-.03 13.45-3.8 11.76-7-.29-.54-.75-.76-1.34-.76-2.38 0-6.71 3.54-8.57 3.54-.42 0-.71-.17-.83-.6-.8-2.85 12.05-4.05 10.97-8.17-.19-.73-.7-1.02-1.44-1.02-3.14 0-10.2 5.53-11.68 5.53-.1 0-.19-.03-.23-.1-.74-1.2-.34-2.04 4.88-5.2 5.23-3.16 8.9-5.06 6.8-7.33-.23-.26-.57-.38-.98-.38-3.18 0-10.67 6.82-10.67 6.82s-2.02 2.1-3.24 2.1a.74.74 0 0 1-.68-.38c-.87-1.46 8.05-8.22 8.55-11.01.34-1.9-.24-2.85-1.31-2.85Z"/><path fill="#FFD21E" d="M56.33 76.69c-2.75-4.04-2.56-7.07 1.22-10.84 3.77-3.77 5.97-9.3 5.97-9.3s.82-3.2 2.7-2.9c1.86.3 3.23 5.08-.68 8.01-3.92 2.93.78 4.92 2.28 2.17 1.51-2.75 5.63-9.82 7.76-11.18 2.13-1.35 3.64-.6 3.13 2.2-.5 2.79-9.42 9.55-8.55 11 .86 1.47 3.92-1.71 3.92-1.71s9.58-8.71 11.66-6.44c2.08 2.27-1.58 4.17-6.8 7.33-5.23 3.16-5.63 4-4.9 5.2.75 1.2 12.28-8.53 13.36-4.4 1.08 4.11-11.76 5.3-10.97 8.15.8 2.85 9.05-5.38 10.74-2.18 1.69 3.21-11.65 6.98-11.76 7.01-4.31 1.12-15.26 3.49-19.08-2.12Z"/></svg>
```

Comment:
Thanks for the pull request @soodaayush!
Please update the icon with the following code. It removes the padding around the icon :wink: 
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 88 81.28"><path fill="#ffd21e" d="M43.75 73.5a34.75 34.75 0 1 0 0-69.5 34.75 34.75 0 0 0 0 69.5Z"/><path fill="#ff9d0b" d="M42.98 0a38.75 38.75 0 0 0-36.2 50.34h-.06c-1.55 0-2.95.58-3.94 1.65a5.8 5.8 0 0 0-.8 7 5.3 5.3 0 0 0-1.79 2.82c-.24.9-.48 2.8.8 4.74a5.22 5.22 0 0 0-.37 5.02c1.02 2.32 3.57 4.14 8.52 6.1 3.07 1.22 5.89 2 5.91 2.01 3.56.99 7.23 1.53 10.93 1.6 5.3 0 9.22-1.47 11.7-4.36 4.16.63 8.38.58 12.52-.14 2.47 2.98 6.44 4.5 11.82 4.5 3.7-.07 7.37-.61 10.93-1.6.02 0 2.84-.79 5.92-2 4.94-1.97 7.5-3.79 8.51-6.1.75-1.63.6-3.53-.37-5.03a5.85 5.85 0 0 0 .8-4.74A5.3 5.3 0 0 0 86.03 59a5.8 5.8 0 0 0-.8-7 5.29 5.29 0 0 0-4.6-1.6 38.75 38.75 0 0 0 1.87-11.64A38.75 38.75 0 0 0 42.98.01Zm.77 4A34.75 34.75 0 0 1 78.5 38.75a34.75 34.75 0 0 1-.97 7.62A5.21 5.21 0 0 0 73.94 45a7.4 7.4 0 0 0-3.96 1.27c-.89.57-1.87 1.56-2.87 2.82a5.7 5.7 0 0 0-4.6-2.46c-3.4 0-5.46 2.96-6.24 5.62-.38.9-2.23 5-5 7.77-4.17 4.16-5.26 8.45-3.3 13.03a35.1 35.1 0 0 1-8 .16c2.04-4.65.98-8.98-3.23-13.19-2.77-2.78-4.62-6.87-5-7.77-.78-2.66-2.84-5.62-6.25-5.62a5.7 5.7 0 0 0-4.6 2.46c-1-1.26-1.98-2.25-2.86-2.82A7.4 7.4 0 0 0 14.06 45a5.2 5.2 0 0 0-4 1.81A34.75 34.75 0 0 1 9 38.75 34.75 34.75 0 0 1 43.75 4Zm-23 23.5c-4.33 0-4.33 6.5 0 6.5 4.34 0 4.34-6.5 0-6.5Zm46.5 0c-4.33 0-4.33 6.5 0 6.5s4.33-6.5 0-6.5Z"/><path fill="#ff323d" d="M43.5 53.3c9.83 0 13-8.77 13-13.27 0-2.34-1.57-1.6-4.09-.36-2.33 1.15-5.46 2.74-8.9 2.74-7.19 0-13-6.88-13-2.38s3.16 13.26 13 13.26Z"/><path fill="#3a3b45" d="M35.97 51a8.7 8.7 0 0 1 5.3-4.49c.4-.12.81.57 1.24 1.28.4.68.82 1.37 1.24 1.37.45 0 .9-.68 1.33-1.35.45-.7.9-1.38 1.32-1.25a8.61 8.61 0 0 1 5 4.17c3.73-2.94 5.1-7.74 5.1-10.7 0-2.34-1.57-1.6-4.09-.36l-.14.07c-2.3 1.15-5.39 2.67-8.77 2.67s-6.45-1.52-8.77-2.67c-2.6-1.29-4.23-2.1-4.23.3 0 3.04 1.46 8.05 5.47 10.96Zm19.07-21.7c1.28.44 1.78 3.06 3.07 2.38a5 5 0 1 0-6.76-2.07c.61 1.15 2.55-.72 3.7-.32Zm-23.55 0c-1.28.44-1.79 3.06-3.07 2.38a5 5 0 1 1 6.76-2.07c-.6 1.15-2.56-.72-3.7-.32Z"/><path fill="#ffd21e" d="M14.11 49c-1.1-.03-1.7.93-1.36 2.85.5 2.8 9.43 9.55 8.56 11-.87 1.47-3.93-1.7-3.93-1.7S7.81 52.42 5.72 54.7c-2.08 2.27 1.6 4.17 6.8 7.33 5.23 3.16 5.64 4 4.9 5.2-.75 1.2-12.28-8.53-13.36-4.4-1.08 4.11 11.77 5.3 10.98 8.15-.8 2.85-9.06-5.38-10.74-2.18-1.7 3.21 11.65 6.98 11.76 7.01 4.3 1.12 15.25 3.5 19.08-2.12 2.75-4.04 2.55-7.07-1.22-10.84-3.78-3.77-5.98-9.3-5.98-9.3s-.82-3.2-2.69-2.9c-1.87.3-3.24 5.08.68 8.01 3.91 2.93-.78 4.92-2.29 2.17-1.5-2.75-5.62-9.82-7.76-11.18a3.64 3.64 0 0 0-1.77-.65Zm59.78 0c-.5.01-1.1.23-1.77.65-2.13 1.36-6.25 8.43-7.76 11.18-1.5 2.75-6.2.76-2.28-2.17 3.91-2.93 2.54-7.7.68-8-1.88-.3-2.7 2.9-2.7 2.9s-2.2 5.52-5.97 9.3c-3.78 3.76-3.97 6.8-1.22 10.83 3.82 5.61 14.77 3.24 19.08 2.12.11-.03 13.45-3.8 11.76-7-1.69-3.2-9.94 5.02-10.74 2.17-.79-2.85 12.05-4.04 10.97-8.15-1.08-4.13-12.61 5.6-13.36 4.4-.73-1.2-.33-2.04 4.9-5.2 5.22-3.16 8.88-5.06 6.8-7.33-2.08-2.27-11.66 6.44-11.66 6.44s-3.06 3.18-3.92 1.71c-.87-1.45 8.05-8.2 8.55-11 .35-1.92-.25-2.88-1.36-2.85Z"/></svg>
```